### ISSUE TITLE
removeView performance

### ISSUE THREAD

**Issue by johan-ohrn** [BorisMoore/jsviews#447]
(I found this issue in the WIP cache version of jquery.views.js but I'm pretty sure it's in the current version as well)

at line 2920 in jquery.views.js:
```
try {
  prevNode.parentNode.removeChild(prevNode); // (prevNode.parentNode is parentElem, except if jQuery Mobile or similar has inserted an intermediate wrapper
  nextNode.parentNode.removeChild(nextNode);
} catch (e) { }
```
Executing this code with one of my templates take ~140ms

```
//try {
  prevNode && prevNode.parentNode.removeChild(prevNode); // (prevNode.parentNode is parentElem, except if jQuery Mobile or similar has inserted an intermediate wrapper
  nextNode && nextNode.parentNode.removeChild(nextNode);
//} catch (e) { }
```
Executing this code with the same templates take ~40ms

prevNode and? nextNode is undefined at times.

**Comment by BorisMoore** [comment]
Yes, you are right about the perf cost, and indeed I think we can simply replace that code by:

```
if (!viewToRemove._elCnt && prevNode) {
	prevNode.parentNode.removeChild(prevNode); // ...
	nextNode.parentNode.removeChild(nextNode);
}

```

**Comment by BorisMoore** [comment]
This has been fixed in new release v1.0.7
